<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>untitled</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="new_btn">
        <property name="styleSheet">
         <string notr="true">background:rgb(170, 85, 255);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="open_btn">
        <property name="styleSheet">
         <string notr="true">background:rgb(0, 170, 255);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="styleSheet">
         <string notr="true">background:rgb(170, 0, 127);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="undo_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 127);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="redo_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 85, 255);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Redo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cut_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 127);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Cut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copy_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 0);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="paste_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);
color:white;
font: 8pt &quot;Calibri&quot;;

</string>
        </property>
        <property name="text">
         <string>Paste</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionDark_Theme"/>
    <addaction name="actionLight_Theme"/>
    <addaction name="actionMonokai_Theme"/>
    <addaction name="actionDracula"/>
    <addaction name="separator"/>
    <addaction name="actionIncrease_Font"/>
    <addaction name="actionDecrease_Font"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionOther_Projects"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDark_Theme">
   <property name="text">
    <string>Dark Theme</string>
   </property>
  </action>
  <action name="actionLight_Theme">
   <property name="text">
    <string>Light Theme</string>
   </property>
  </action>
  <action name="actionMonokai_Theme">
   <property name="text">
    <string>Monokai Theme</string>
   </property>
  </action>
  <action name="actionDracula">
   <property name="text">
    <string>Dracula</string>
   </property>
  </action>
  <action name="actionIncrease_Font">
   <property name="text">
    <string>Increase Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionDecrease_Font">
   <property name="text">
    <string>Decrease Font</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionOther_Projects">
   <property name="text">
    <string>Other Projects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
